src/base/bittorrent/torrentcreatorthread.cpp

```diff
@@ -133,8 +133,13 @@ void TorrentCreatorThread::run()
 
         if (isInterruptionRequested()) return;
 
+#if LIBTORRENT_VERSION_NUM < 10100
```

Comment:
I was writing the patch too! there is only one thing different... I added `#include<libtorrent/version.hpp>` in this file, with out this include, it will use the deprecated `create_torrent::optimize` on RC_1_1.

Reply:
You're correct. I totally forgot about that. I pushed an updated commit.